### PR TITLE
Adds a 'Getting Started' guide to Vocabulary

### LINKED ISSUE #377: Demonstration of Vocabulary Usage
**Description**

To make it easier for a user who is unfamiliar with vocabulary, a demonstration of using vocabulary components with a basic site created can be made.

**Problems solved**

Users will not have to spend time understanding vocabulary.

**Alternatives**

**Additional context**

**Want to take up?**

Yes

### PR DESCRIPTION
**Pull request by nimishbongale**
## Fixes
Fixes #377 by @jahnvigupta 

## Description
Adds a Getting Started story to our Vocabulary storybook as a part of a series 'Usage Guide' additions (GSoD '20).

I've gone ahead and used [JSFiddle](https://jsfiddle.net/), but we can totally make a group of codepens and link the same as well!

## Screenshot

ss

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by nimishbongale** [comment]
@zackkrida 

I just resolved a merge conflict in this PR. It is ready to merge as well; just that I'm a tad bit unsure why the vue-vocabulary build fails here 🤔

File: packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+<Meta title="Vocabulary/Getting Started" />
+
+# Getting Started
+
+We hope you've grasped what `Vocabulary` is all about. Let's now dive straight into building a simple web application with it.
+
+## Preface
+
+You've learnt how to install `Vocabulary` and the various ways of doing the same in the previous section. You're now _almost_ ready to build your first `CC Vocabulary` powered application!
```
Comment (zackkrida) [inline packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx]:
```suggestion
You've learnt multiple ways to install `Vocabulary` in the previous section. You're now _almost_ ready to build your first `CC Vocabulary` powered application!
```

Made this sentence a little simpler 👍

File: packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+Throughout this tutorial, you will learn the _"how to's"_ when it comes to using our components. It's really not that difficult, we promise!
```
Comment (zackkrida) [inline packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx]:
```suggestion
Throughout this tutorial, you will learn the _"how-tos"_ when it comes to using our components. It's really not that difficult, we promise!
```

**Review by zackkrida — APPROVED** [review]
This is a really nice resource for our users. I rebased and force-pushed to fix the `vue-vocabulary` build issue. That was a downstream issue from another PR.

**Pull request merged**